bin/vgac2pps.py

```diff
@@ -46,10 +46,9 @@
     parser.add_argument('-all_ch', '--all_channels', action='store_true',
                         help="Save all 21 channels to level1c4pps file.")
     parser.add_argument('-n19', '--as_noaa19',
-                        choices=[version for version in SBAF],
                         default=None,
-                        help=("Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. "
-                              "And apply SBAFs to the channels."))
+                        help=(f"Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. "
+                              f"And apply SBAFs to the channels. The SBAF options are {[version for version in SBAF]}"))
```

Comment:
The choices you have are already displayed in the help message by default, so not needed to be listed twice.

Reply:
What I mean is that the help message below describes the choices you have. To me it just adds noise if you add them also in the description.
```shell
$ python bin/vgac2pps.py --help
usage: vgac2pps.py [-h] [-o [OUT_DIR]] [-ne [NC_ENGINE]] [-all_ch] [-n19 {v2,v3,v4,v5,v6}] [-pps_ch] [-avhrr_ch] [-on [ORBIT_NUMBER]] [--don_split_files_at_midnight] fileN [fileN ...]

Script to produce a PPS-level1c file for a VIIRS level-1 scene

positional arguments:
  fileN                 List of VIIRS files to process

optional arguments:
  -h, --help            show this help message and exit
  -o [OUT_DIR], --out_dir [OUT_DIR]
                        Output directory where to store the level1c file
  -ne [NC_ENGINE], --nc_engine [NC_ENGINE]
                        Engine for saving netcdf files netcdf4 or h5netcdf (default).
  -all_ch, --all_channels
                        Save all 21 channels to level1c4pps file.
  -n19 {v2,v3,v4,v5,v6}, --as_noaa19 {v2,v3,v4,v5,v6}
                        Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. And apply SBAFs to the channels.
  -pps_ch, --pps_channels
                        Save only the necessary (for PPS) channels to level1c4pps file.
  -avhrr_ch, --avhrr_channels
                        Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file.
  -on [ORBIT_NUMBER], --orbit_number [ORBIT_NUMBER]
                        Orbit number (default is 00000).
  --don_split_files_at_midnight
                        Don't split files at midnight, keep as one level1c file.
```

Reply:
And if the help message throw an error for you, you should fix it :). I think this is not covered by any of the current test.

Reply:
But where does that help message come from?

Reply:
from the "argparse" library.

Reply:
OK. Long story short. I'll remove the SBAF version info from the help argument